extensions/arc/images/gear-black.svg

```diff
@@ -0,0 +1,3 @@
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 16 16">
```

Comment:
How does this look on dark theme? It would be nice to have separate icons regardless - I can get you in touch with our designers to get some icons if you don't have one in mind

Reply:
This one does not show up well but was set black to follow the design on portal. An alternate would be helpful.

Reply:
will be changed to switch to white gear in darker themes